pkg/controller/manage/kmesh_manage.go

```diff
@@ -54,7 +54,7 @@ var (
 )
 
 const (
-	DefaultInformerSyncPeriod = 2 * time.Second
+	DefaultInformerSyncPeriod = 30 * time.Second
```

Comment:
Is there no way to optimise it?

Reply:
Synchronization here shouldnot be too frequent, and the interval should be as long as possible. In fact, if there are changes, they will be pushed over immediately, and there is no need to refresh them here regularly.

Reply:
I would suggest removing resync https://github.com/kmesh-net/kmesh/issues/597